### DISCUSSION TITLE
v1.1.0-preview3

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#2463] [Announcements]
### 1.1 Preview 3 Release of Windows App SDK 

We've just released our third preview of the 1.1 release of the Windows App SDK! This preview includes new features and bug fixes, including:
- Mica and Background Acrylic available for WinUI 3 apps
- New extension categories in MSIX
- Several bug fixes in the Notifications area

To see everything that's new and changed, see the full [release notes here](https://aka.ms/windowsappsdk/1.1/1.1.0-preview3/release-notes). 

### What's next 
To keep up with progress being made on the Windows App SDK, please see our [feature roadmap](https://aka.ms/winappsdkroadmap), which gets updated regularly.  

### Providing feedback
As we introduce new & upcoming features in our preview releases, we appreciate all of your feedback as we work towards a stable release:
- For WinUI 3 feedback, please open an issue on the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues/new/choose).
- For feedback regarding all other Windows App SDK components, please open an issue here, on the [Windows App SDK repo](https://github.com/microsoft/WindowsAppSDK/issues/new/choose).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.1.0-preview3'>v1.1.0-preview3</a>.</em>

**Comment by DominicMaas** [comment]
Will Background Acrylic make its way back down to Windows 10? (`DesktopAcrylicController.IsSupported()` is returning false on my Windows 10 machine).

**Comment by Samuel12321** [comment]
@gabbybilka Can you clarify whether background acrylic in winui 3 is supported on Windows 10

**Comment by AathifMahir** [comment]
@gabbybilka Any update on Built in Splash Screen Support Like UWP for WinUI 3? on UWP we had Consistent Splash Screen Across the board with Flexibility of Implementing Custom Splash Screen as Well, that made UWP apps to be on a standard and consistent, Now with WinUI 3 we don't have built in Splash Screen Support that Causes Many WinUI 3 App to be Non-Consistent at Startup and Many WinUI 3 Apps Implements Custom Splash Screens that were in Different Standards by Apps

**Reply by castorix** [reply]
> @gabbybilka Any update on Built in Splash Screen Support Like UWP for WinUI 3? on UWP we had Consistent Splash Screen Across the board with Flexibility of Implementing Custom Splash Screen as Well, that made UWP apps to be on a standard and consistent, Now with WinUI 3 we don't have built in Splash Screen Support that Causes Many WinUI 3 App to be Non-Consistent at Startup and Many WinUI 3 Apps Implements Custom Splash Screens that were in Different Standards by Apps

Splash Screens are not too difficult to implement.
I made a basic sample by using a Layered window : [WinUI3_SplashScreen](https://github.com/castorix/WinUI3_SplashScreen)
I added an animation (Fade-Out) on window closing,  but you can add other functionalities and improve it.

**Reply by AathifMahir** [reply]
> > @gabbybilka Any update on Built in Splash Screen Support Like UWP for WinUI 3? on UWP we had Consistent Splash Screen Across the board with Flexibility of Implementing Custom Splash Screen as Well, that made UWP apps to be on a standard and consistent, Now with WinUI 3 we don't have built in Splash Screen Support that Causes Many WinUI 3 App to be Non-Consistent at Startup and Many WinUI 3 Apps Implements Custom Splash Screens that were in Different Standards by Apps
> 
> Splash Screens are not too difficult to implement.
> I made a basic sample by using a Layered window : [WinUI3_SplashScreen](https://github.com/castorix/WinUI3_SplashScreen)
> I added an animation (Fade-Out) on window closing,  but you can add other functionalities and improve it. 

I also developed one for my WinUI 3 app that currently in development, I wasn't saying it's hard to create splash screen instead I asking about splash screen like UWP that baked in, instead custom splash screen would be only the option, The main takeaway here is consistency on splash screen, UWP had Consistent Splash Screen Design that Many Developers were adapted but that wasn't the case with WinUI 3 yet
